### PR TITLE
Added Simplenote version 2.16.0.

### PR DESCRIPTION
**Pull request by jedieaston**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

------

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24330)

### CODE REVIEW

**Comment by jedieaston** [comment]
image

It works fine for me. Is there a way to see which installer is failing?

**Comment by palenshus** [comment]
This is failing due to a SignatureSha256 hash mismatch

File: manifests/a/Automattic/Simplenote/2.16.0/Automattic.Simplenote.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- silent
+Installers:
+- Platform:
+  - Windows.Universal
+  Architecture: x64
+  InstallerType: appx
+  Scope: user
+  InstallerUrl: https://github.com/Automattic/simplenote-electron/releases/download/v2.16.0/Simplenote-win-2.16.0-x64.appx
+  InstallerSha256: DF80EE34626BFA15A7F27C8CAC4F4F32B275FC876A05708A826F54C682F34B64
+  SignatureSha256: 0B949D4FBC8FE97553562F2678E86A2CAA6AF8FBDA26CA954A6AE4DF785AA653
```
Comment (ItzLevvie) [inline manifests/a/Automattic/Simplenote/2.16.0/Automattic.Simplenote.installer.yaml:16]:
```suggestion
  SignatureSha256: C1D6E63B2F91F7CE83BE5B9FE9E2C75055810A9DDEA3B69866AA5FD377948EAD
```

File: manifests/a/Automattic/Simplenote/2.16.0/Automattic.Simplenote.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  PackageFamilyName: simplenote_cdagv53c87npa
+  UpgradeBehavior: install
+- Platform:
+  - Windows.Universal
+  Architecture: x86
+  InstallerType: appx
+  Scope: user
+  InstallerUrl: https://github.com/Automattic/simplenote-electron/releases/download/v2.16.0/Simplenote-win-2.16.0-ia32.appx
+  InstallerSha256: B0DD55DE247FBA40626CE349B7E777C4B1B9CB4378FD191E14BCC07030F0B97A
+  SignatureSha256: 01C0122D12D272C021C1E0A50B91FC8BF8CD6CF23007752C5F50EC3C2267B907
```
Comment (ItzLevvie) [inline manifests/a/Automattic/Simplenote/2.16.0/Automattic.Simplenote.installer.yaml:26]:
```suggestion
  SignatureSha256: ECEDFBB2B41999026870C45D054168F5F1FF6FB20F75AC2299F9776CFB95EF9C
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by jedieaston** [comment]
@palenshus Does winget not verify the SignatureSha256 on `winget install` right now? It installed for me (see screenshot above).

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**